brainglobe_atlasapi/atlas_generation/atlas_scripts/praire_vole.py

```diff
@@ -0,0 +1,256 @@
+        print(f"Files: {file_list}")
+        zip_ref.extract(
+            "results_datasets/neural_nomenclature_hierarchy.csv", atlas_path
+        )
+
+    shutil.move(
+        atlas_path / r"results_datasets\neural_nomenclature_hierarchy.csv",
+        atlas_path,
+    )
+    os.rmdir(atlas_path / r"results_datasets")
+
+    annotations_file = atlas_path / "PrV_Annotation_bilateral.tif"
+    reference_file = atlas_path / "PrV_ReferenceBrain_bilateral.tif"
+    meshes_dir_path = atlas_path / "meshes"
+    meshes_dir_path.mkdir(exist_ok=True)
+
+    nmh = pd.read_csv(atlas_path / "neural_nomenclature_hierarchy.csv").values
+    reference_id = pd.read_csv(atlas_path / "PrV_Atlas_Ontology.csv")
+
+    acronyms = reference_id["acronym"].values
+    names = reference_id["name"].values
+    ids = reference_id["id"].values
+
+    acronym_to_id = dict(zip(acronyms, ids))
+    name_to_id = dict(zip(names, ids))
+
+    ids = list(ids)
+    hierarchy = [
+        {
+            "acronym": "root",
+            "id": 997,
+            "name": "root",
+            "structure_id_path": [997],
+            "rgb_triplet": [255, 255, 255],
+        }
+    ]
+
+    for row in nmh:
+        structure_hierarchy = []
+        unique_pairs = []
+        pairs = [(row[i], row[i + 1]) for i in range(0, len(row), 2)]
+        pairs_seen = set()
+
+        for pair in pairs:
+            if pair not in pairs_seen:
+                unique_pairs.append(pair)
+                pairs_seen.add(pair)
+
+        for acronym, name in unique_pairs:
+            if acronym in acronym_to_id:
+                structure_hierarchy.append(acronym_to_id[acronym])
+            elif name in name_to_id:
+                structure_hierarchy.append(name_to_id[name])
+
+        structure_template = {
+            "acronym": acronyms[ids.index(structure_hierarchy[-1])],
+            "id": structure_hierarchy[-1],
+            "name": names[ids.index(structure_hierarchy[-1])],
+            "structure_id_path": structure_hierarchy,
+            "rgb_triplet": [random.randint(0, 255) for _ in range(3)],
+        }
+
+        hierarchy.append(structure_template)
+
+    # use tifffile to read annotated file
+    annotated_volume = tifffile.imread(annotations_file).astype(np.uint8)
+    reference_volume = tifffile.imread(reference_file)
+
+    print(f"Saving atlas data at {atlas_path}")
+    print(hierarchy)
+
+    tree = get_structures_tree(hierarchy)
+    print(
+        f"Number of brain regions: {tree.size()}, "
+        f"max tree depth: {tree.depth()}"
+    )
+    labels = np.unique(annotated_volume).astype(np.int32)
+    for key, node in tree.nodes.items():
+        if key in labels:
+            is_label = True
+        else:
+            is_label = False
+        node.data = Region(is_label)
+
+    # Mesh creation
+    closing_n_iters = 2
+    decimate_fraction = 0.2
+    smooth = False  # smooth meshes after creation
+    start = time.time()
+    if PARALLEL:
```

Comment:
Our parallel mesh extraction is broken, so all of this part can be removed, and just keep the stuff after `else:`.